tracer/src/Datadog.Trace/Debugger/Symbols/SymbolExtractor.cs

```diff
@@ -691,7 +690,7 @@ void PopulateClosureMethod(MethodDefinition generatedMethod, TypeDefinition owne
             foreach (var parameterHandle in parameters)
             {
                 var parameterDef = MetadataReader.GetParameter(parameterHandle);
-                if (index == 0)
+                if (index == 0 && !method.IsStaticMethod())
```

Comment:
Maybe add a test that covers this change and the compiler generated one?

Reply:
You absolutely right about it. I'll add test in next PR that will include also PDB absent assemblies.